database/migrations/1733797260473-fixed-booking-status.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class FixedBookingStatus1733797260473 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TYPE "booking_status_enum"
+      ADD VALUE 'cancelled';
```

Comment:
Please use the american version ```canceled```. In other parts of the project we are using it. Let's keep consistency.

Reply:
Fixed